plugin/pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+# name = "bgutil-ytdlp-pot-provider"
+name = "definitely_not_bgutil-ytdlp-pot-provider_to_distinguish_from_it"
```

Comment:
Just remember to fix this before we merge

Reply:
yeah it's in the task list
> * [ ]  Change `project.name` back to normal in `plugin/pyproject.toml` (needs changes in the repo)
> * [ ]  Change `project.version` to the normal one in `plugin/pyproject.toml` (needs changes in the repo)